sources/exegol/load_supported_setups.sh

```diff
@@ -106,6 +106,56 @@ function run_user_setup() {
   echo "[$(date +'%d-%m-%Y_%H-%M-%S')] ==== End of custom setups loading ===="
 }
 
+function deploy_firefox_addons() {
+  ##### firefox custom addons deployment
+  if [ -d "$MY_Setup_PATH/firefox/" ]; then
+    if [ -d "$MY_Setup_PATH/firefox/addons" ]; then
+      ADDON_FOLDER="-D $MY_Setup_PATH/firefox/addons"
+    else
+      mkdir "$MY_Setup_PATH/firefox/addons" && chmod 770 "$MY_Setup_PATH/firefox/addons"
+    fi
+    if [ -f "$MY_Setup_PATH/firefox/addons.txt" ]; then
+      ADDON_LIST="-L $MY_Setup_PATH/firefox/addons.txt"
+    else
+      cp --preserve=mode /.exegol/skel/firefox/addons.txt "$MY_Setup_PATH/firefox/addons.txt"
+    fi
+    python3 /opt/tools/firefox/user-setup.py $ADDON_LIST $ADDON_FOLDER
+  else
+    mkdir --parents "$MY_Setup_PATH/firefox/addons" && chmod 770 -R "$MY_Setup_PATH/firefox/addons"
+    cp --preserve=mode /.exegol/skel/firefox/addons.txt "$MY_Setup_PATH/firefox/addons.txt"
+  fi
+}
+
+function trust_ca_burp() {
+  ##### Burp CA trust
+  if ! [ -d "$MY_Setup_PATH/burp/" ]; then
+    mkdir "$MY_Setup_PATH/burp" && chmod 770 "$MY_Setup_PATH/burp"
+  fi
+  if [ -f "$MY_Setup_PATH/burp/cacert.der" ]; then
+    BURP_CA_PATH="$MY_Setup_PATH/burp/cacert.der"
+    certutil -A -n "PortSwigger CA" -t "TC" -i $BURP_CA_PATH -d ~/.mozilla/firefox/*.Exegol
+  else
+    if [ -d "/opt/tools/BurpSuiteCommunity/" ]; then
+      BURP_PORT=8080
+      LISTENING_PORTS=$(netstat -lnt|grep -Eo '(127.0.0.1|0.0.0.0):[0-9]{1,5}'|cut -d ':' -f 2)
+      while [[ $LISTENING_PORTS =~ .*$BURP_PORT.* ]]
+      do
+        BURP_PORT=$((BURP_PORT+1))
+      done
+      sed -i "s/\"listener_port\":[0-9]\+/\"listener_port\":$BURP_PORT/g" /opt/tools/BurpSuiteCommunity/conf.json
+      echo y|java -Djava.awt.headless=true -jar /opt/tools/BurpSuiteCommunity/BurpSuiteCommunity.jar --config-file=/opt/tools/BurpSuiteCommunity/conf.json 1>/tmp/burp_logs &
+      while ! [[ $(tail /tmp/burp_logs) =~ .*y/n.* ]]
+      do
+          sleep 0.5
+      done
+      BURP_PID=$! && sleep 10 && wget "http://127.0.0.1:$BURP_PORT/cert" -O /tmp/cacert.der && kill $BURP_PID
+      BURP_CA_PATH="/tmp/cacert.der"
+      certutil -A -n "PortSwigger CA" -t "TC" -i $BURP_CA_PATH -d ~/.mozilla/firefox/*.Exegol
```

Comment:
When the CA is generated & sign ? 
This command import the CA to Firefox right ?

Reply:
Case of Exegol's BurpSuiteCommunity :
The CA is generated when Burp is opened.
It is then downloaded to /tmp/cacert.der from where it is trusted in Firefox via certutil.

Case of host's BurpSuite :
The cacert.der in my-resources is trusted in Firefox via certutil.